Dockerfile

```diff
@@ -9,10 +9,33 @@ LABEL about.license="MIT License (MIT)"
 # Install required libs
 RUN apk update
 RUN apk --no-cache add make automake gcc g++ linux-headers libffi-dev zlib-dev \
-   jpeg-dev libressl-dev cairo-dev pango-dev gdk-pixbuf ttf-freefont bash
+   jpeg-dev libressl-dev cairo-dev pango-dev gdk-pixbuf ttf-freefont bash \
+   autoconf musl-dev perl bzip2-dev xz-dev curl-dev
 RUN pip install numpy Cython
 
+# Install cyvcf2
+RUN apk add git
+WORKDIR /tmp
+RUN git clone --recursive https://github.com/brentp/cyvcf2
+WORKDIR /tmp/cyvcf2/htslib
+RUN autoheader
+RUN autoconf
+RUN ./configure --enable-libcurl
+RUN make
+WORKDIR /tmp/cyvcf2
+RUN pip install -r requirements.txt
+RUN CYTHONIZE=1 pip install -e .
```

Comment:
I had seen this solution [here](https://github.com/brentp/cyvcf2/pull/143#issuecomment-606077539) as well. It means recompile Cython? Then it should install it on clinical-db right?

Reply:
We would probably need to change the scripts of clinical-db to have CYTHONIZE=1 as well right?

Reply:
That might actually just work; it still needs htslib, which I thought was the issue here (needs a --recursive pull from git, as it is only linked). Actually now that I think about it, we should probably change to a versioned release as well - it's all good to use cyvcf2-master for development, but maybe not ideal if this turns into the preferred deployment method

Reply:
For reference, this gets stuck wanting `bzip2-devel` on clinical-db. Not insurmountable, but tedious and complex since it is not available prepackaded from conda. (Also, the autoconf result is a little weird; looks like autotools are  not quite perfectly set up either?)